cmd/aro/update_ocp_versions.go

```diff
@@ -34,57 +39,111 @@ func getInstallerImageDigests(envKey string) (map[string]string, error) {
 	if !env.IsLocalDevelopmentMode() {
 		jsonData, err = base64.StdEncoding.DecodeString(string(jsonData))
 		if err != nil {
-			return nil, fmt.Errorf("%s: Failed to decode base64: %v", envKey, err)
+			return fmt.Errorf("%s: Failed to decode base64: %v", envKey, err)
```

Comment:
scope creep, I know... Feel free to :hear_no_evil: and we can groom additional work...

...but while we're touching this, can we maybe make these log messages something that would be useful in production, even adding a bit of context like "Failed to base64 decode install image digests" and use `%w` instead of `%v`?